content/client-lib-development-guide/features.textile

```diff
@@ -301,12 +301,12 @@ h3(#realtime-connection). Connection
 * @(RTN3)@ If connection option @autoConnect@ is true, a connection is initiated immediately; otherwise a connection is only initiated following an explicit call to @connect()@
 * @(RTN4)@ EventEmitter and states:
 ** @(RTN4a)@ Implements @EventEmitter@ and emits events for state changes @INITIALIZED@, @CONNECTING@, @CONNECTED@, @DISCONNECTED@, @SUSPENDED@, @CLOSING@, @CLOSED@, @FAILED@
```

Comment:
Should we not just mention this is a `ConnectionEvent` type that extends `ConnectionState` and adds `UPDATE` event?